content/doc/book/updating/windows.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+title: Windows
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:left
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Windows
+
+There are two ways to upgrade Jenkins in Windows.
+One is via Windows MSI installer
+Other one is discussed later on the page.
```

Comment:
```suggestion
Other one is discussed later on the page.
```
Instead of saying one is discussed later, say what the name of it is here and link to it. This information does not help the user how it is written.